jwst/cube_build/__init__.py

```diff
@@ -1,4 +1,4 @@
 from __future__ import absolute_import
 from .cube_build_step import CubeBuildStep
 
-__version__ = '0.8.0'
+__version__ = '0.9.0'
```

Comment:
Might as well go ahead and set that to '0.9.3' in anticipation of the B7.1.3 delivery this week.